packages/module/src/Feedback/Feedback.scss

```diff
@@ -69,20 +69,20 @@
 }
 
 .chr-c-feedback-email {
-  margin-top: var(--pf-v5-global--spacer--lg);
+  margin-top: var(--pf-t--global--spacer--lg);
   overflow: hidden;
 }
 
 .chr-c-feedback-image {
-  background-color: var(--pf-v5-global--palette--black-150);
+  background-color: var(--pf-t--global--background--color--secondary--default);
```

Comment:
This color wasn't an exact match (f2f2f2 vs f5f5f5) but I didn't see a token value that matched black-150 exactly. Is this change acceptable, or should I have used a different token? Or just left it be.

cc @andrew-ronaldson @lboehling